perl/README.md

```diff
@@ -0,0 +1,1029 @@
+# Bystro High Dimensional Genomic Annotator Documentation
+
+## What is the Bystro Annotator?
+
+In order to make use of genetic data, which is composed of variants (colloquially known as mutations) and the individuals (samples) that have those variants, we need to clean that data and then carefully describe those variants with as much accurate information as possible. This information can then be used for qualitative and quantitative studies of the variants/samples. These descriptions are known as labels, model parameters, features, or "annotations", depending on your field. These activites are collectively called data curation and data labeling, but in genetics we call them qc and annotation.
+
+The Bystro Annotator is the fastest and most comprehensive data curation and labeling library in the world for genetic data. It takes 1 or more VCF ([Variant Call Format](https://samtools.github.io/hts-specs/VCFv4.2.pdf)) or SNP ([PEMapper/PECaller](https://www.pnas.org/doi/full/10.1073/pnas.1618065114)) files as input, and outputs a cleaned and thoroughly labeled (annotated) representation of the data, along with a genotype dosage matrix in the [Arrow Feather V2/IPC format](https://arrow.apache.org/docs/python/feather.html), as well as a set of statistics that describe sample-level characteristics of the data.
+
+Bystro Annotator annotates variants as well as sample genotypes. It is capable of processing millions of samples and billions of mutations on commodity hardware such as a laptop or a workstation. It is roughly **100,000** times faster than [Variant Effect Predictor](https://www.ensembl.org/info/docs/tools/vep/index.html) (VEP), **100** times faster than [Annovar](https://doc-openbio.readthedocs.io/projects/annovar/en/latest/), and **50** times faster than [Illumina Connected Annotations](https://developer.illumina.com/illumina-connected-annotations), all while outputting more annotations than any of these tools. What would take VEP years to do, Bystro can do in minutes to hours, all without requiring multiple servers.
```

Comment:
VEP and Annovar links seem to no longer be active

Reply:
fixed